api/src/trace/span.ts

```diff
@@ -125,5 +125,9 @@ export interface Span {
    * @param [time] the time to set as Span's event time. If not provided,
    *     use the current time.
    */
-  recordException(exception: Exception, time?: TimeInput): void;
+  recordException(
```

Comment:
This signature change looks like a breaking change to me.

Reply:
Ok, looking at it then this "really" would allow the usage of recordException with 2 timeInputs.

Suggest: defining 2 explicit signatures in the interface to call out the expectation.  And document that when `attributesOrStartTime` is a time that any 3rd argument is ignored and/or the call will fail.

Reply:
Also need to add the param to the documentation

Reply:
I'd generally like it too, and suggested as much in the https://github.com/open-telemetry/opentelemetry-js/issues/4070. However, this would be different from the `addEvent()` API. And as you already noted, changing `addEvent()` now would be a breaking change. Do you think it's better to make a change here or keep consistent with the `addEvent()`?

Reply:
I updated this API as suggested, but kept the `addEvent` unchanged to avoid breaking changes.